smartsim/_core/launcher/dragon/dragonBackend.py

```diff
@@ -288,6 +291,29 @@ def current_time(self) -> float:
         """Current time for DragonBackend object, in seconds since the Epoch"""
         return time.time()
 
+    def _can_honor_policy(
+        self, request: DragonRunRequest
```

Comment:
Can we add the proper doc strings to this method

Reply:
done!